client.go

```diff
@@ -32,14 +32,20 @@ type HTTPError struct {
 	Status string
 }
 
+// Result is the result of a SimpleResponse
+type Result string
+
 // SimpleResponse is a structure that returns success and/or any error
 type SimpleResponse struct {
-	Result       string `json:"result"`
+	Result       Result `json:"result"`
```

Comment:
Bearing in mind I'm not an expert Go programmer - what's the reason for using a custom type here over a simple string?

Reply:
I like to use types in these situations to signal that the API won't be returning just any string, but from a specific subset. Similar to what you'd do with an `enum` in other languages.